app/Database/migrations/2024_12_03_093851_add_sources_to_acars_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('acars', function (Blueprint $table) {
+            $table->integer('source')->nullable();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::table('acars', function (Blueprint $table) {
+            //
```

Comment:
If you are doing nothing why having `down()` function? You can simply delete that complete block :)

Reply:
Fixed